### PR TITLE
Checkstyle and find bugs in state and spaceinvaders package

### PR DESCRIPTION
**Pull request by Mari3**

### CODE REVIEW

File: src/main/java/spaceinvaders/Game.java [hunk 1]
```diff
@@ -40,7 +38,11 @@
   private Executor exec;
 
   /**
```
Comment (JerPk) [inline src/main/java/spaceinvaders/Game.java:40]: You forgot to remove the merge tags

File: src/main/java/spaceinvaders/LogFile.java [hunk 2]
```diff
@@ -97,19 +97,20 @@ public boolean writeCreate(String object, double x, double y) {
   }
 
   /**
-   * method to write object which shot to logfile
+<<<<<<< HEAD
```
Comment (JerPk) [inline src/main/java/spaceinvaders/LogFile.java]: Another forgotten merge tag

File: src/main/java/spaceinvaders/LogFile.java [hunk 3]
```diff
@@ -97,19 +97,20 @@ public boolean writeCreate(String object, double x, double y) {
...
+   * method to write object which shot to logfile.
    * 
    * @param object
    *          which shot
-   * @param x
+   * @param xco
    *          position of the object
-   * @param y
+   * @param yco
```
Comment (JerPk) [inline src/main/java/spaceinvaders/LogFile.java:106]: I think xco and yco aren't that clear, maybe xpos/ypos or xcoord/ycoord?

File: src/main/java/spaceinvaders/LogFile.java [hunk 4]
```diff
@@ -118,19 +119,19 @@ public boolean writeShoot(String object, double x, double y) {
...
-   * method to write moved object to logfile
+   * method to write moved object to logfile.
    * 
    * @param object
    *          which has moved
-   * @param x
+   * @param xco
    *          position of the object
-   * @param y
+   * @param yco
```
Comment (JerPk) [inline src/main/java/spaceinvaders/LogFile.java:127]: I think xco and yco aren't that clear, maybe xpos/ypos or xcoord/ycoord?

File: src/main/java/spaceinvaders/LogFile.java [hunk 5]
```diff
@@ -139,19 +140,19 @@ public boolean writeMove(String object, double x, double y) {
...
-   * method to write hit object to logfile
+   * method to write hit object to logfile.
    * 
    * @param object
    *          which is hit
-   * @param x
+   * @param xco
    *          position of the object
-   * @param y
+   * @param yco
```
Comment (JerPk) [inline src/main/java/spaceinvaders/LogFile.java:148]: I think xco and yco aren't that clear, maybe xpos/ypos or xcoord/ycoord?

File: src/main/java/spaceinvaders/LogFile.java [hunk 6]
```diff
@@ -160,16 +161,16 @@ public boolean writeHit(String object, double x, double y) {
   }
 
   /**
-   * method to write object which went offscreen to logfile
+   * method to write object which went offscreen to logfile.
    * 
    * @param object
    *          which went off the screen
-   * @param x
+   * @param xco
```
Comment (JerPk) [inline src/main/java/spaceinvaders/LogFile.java:167]: I think xco isn't that clear, maybe xpos  or xcoord?

File: src/main/java/spaceinvaders/Menu.java [hunk 7]
```diff
@@ -31,40 +26,50 @@
...
+   *          executor for the state pattern
+   */
   public Menu(Executor ex) {
     running = false;
     exec = ex;
 
     JPanel panel = new JPanel();
     panel.setLayout(new GridBagLayout());
-    GridBagConstraints c = new GridBagConstraints();
+    GridBagConstraints gbc = new GridBagConstraints();
```
Comment (JerPk) [inline src/main/java/spaceinvaders/Menu.java]: Maybe better to change this to constraints (as I already did for the transition screen)

File: src/main/java/spaceinvaders/ScoreMenu.java [hunk 8]
```diff
@@ -32,9 +31,16 @@
   private Boolean running;
   private Thread thread;
   private Executor exec;
-  private GridBagConstraints c;
+  
+  private GridBagConstraints gbc;
```
Comment (JerPk) [inline src/main/java/spaceinvaders/ScoreMenu.java]: gbc -> constraints

**Comment by JerPk** [comment]
Great improvements and comments everywhere!
Can be merged after you looked at my inline comments.

**Comment by Mari3** [comment]
I think xco and yco is clear enough but I'll change gbc

**Comment by harveyvanveltom** [comment]
looks good now. can be merged.

**Pull request merged**